src/libs/actions/User.ts

```diff
@@ -12,84 +11,93 @@ import * as ReportActionsUtils from '@libs/ReportActionsUtils';
 import CONST from '@src/CONST';
 import ONYXKEYS from '@src/ONYXKEYS';
 import ROUTES from '@src/ROUTES';
+import type {FrequentlyUsedEmoji} from '@src/types/onyx';
+import type Login from '@src/types/onyx/Login';
+import {OnyxServerUpdate} from '@src/types/onyx/OnyxUpdatesFromServer';
+import type OnyxPersonalDetails from '@src/types/onyx/PersonalDetails';
+import ReportAction from '@src/types/onyx/ReportAction';
 import * as Link from './Link';
 import * as OnyxUpdates from './OnyxUpdates';
 import * as PersonalDetails from './PersonalDetails';
 import * as Report from './Report';
 import * as Session from './Session';
 import redirectToSignIn from './SignInRedirect';
 
-let currentUserAccountID = '';
+type CustomStatus = {text: string; emojiCode: string; clearAfter?: string};
+type BlockedFromConciergeNVP = {expiresAt: number};
+
+let currentUserAccountID = -1;
```

Comment:
`currentUserAccountID` is used in `if` conditions in this file, since you replaced `''` value with `-1` it can affect that conditions
[Example](https://github.com/Expensify/App/blob/cba3f8cd93bdfbdc0776a1ac18aeecabd83f7a9e/src/libs/actions/User.ts#L483C6-L483C6)